### ISSUE TITLE
Error in nextTick: "TypeError: Failed to execute 'insertBefore' on 'Node': parameter 1 is not of type 'Node'."

### ISSUE THREAD

**Issue by rangeldor** [lusaxweb/vuesax-next#128]
Tooltip has an error when I change the route, however he worked normally.

**Comment by brdtheo** [comment]
I had the same issue related to a `vs-dialog`component. The fix was to wrap it into a div and add a `v-show`.
However i wonder what could fix the issue on a tooltip 😕

**Comment by rangeldor** [comment]
How can I fix it?

**Comment by brdtheo** [comment]
What if you try to remove tooltips from the DOM after you changed the route ?

```js
router.afterEach(() => {
  const tooltip = document.querySelector('.vs-tooltip')
  if (tooltip !== null) {
    document.body.removeChild(tooltip);
  }
})
```

**Comment by rangeldor** [comment]
Thank you, but he never takes the vs.tooltip. Probrabily because the vs-tootltip is in a V-IF.

**Comment by brdtheo** [comment]
could we see that part of the code ?

**Comment by rangeldor** [comment]
Of course! However, there are a lot of vs-tooltip in that part of the project and I don't know which one of them has a problem, because this problem happens sometime. 

![image](https://user-images.githubusercontent.com/51749346/107236763-c3a95500-6a04-11eb-87b9-04ce6b27244f.png)

**Comment by brdtheo** [comment]
It seems like you have the problem i had but my fix is not working for you 😕
That's maybe a `v-if` related problem as you said ?

**Comment by rangeldor** [comment]
Okay. Thanks for trying to help me.

**Comment by rangeldor** [comment]
Thank you, but the pull request will be accept?
